lxd/db/images.go

```diff
@@ -456,11 +457,11 @@ func (c *Cluster) GetImage(fingerprintPrefix string, filter ImageFilter) (int, *
 	var image api.Image
 	var object Image
 	if fingerprintPrefix == "" {
-		return -1, nil, ErrNoSuchObject
+		return -1, nil, errors.New("No fingerprint prefix specified for the image")
```

Comment:
Why not use `http.StatusNotFound` here as well?

Reply:
In my view the original ErrNoSuchObject was incorrect, as it was not specified, and was more like the error below when no project is specified.